plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -445,69 +447,52 @@ private void createAndPopulateMixedCaseHybridTablesAndTopic(TestingKafka kafka,
                         .set("longCol", 2L)
-                        .set("updatedAt", initialUpdatedAt.plusMillis(2000).toEpochMilli())
+                        .set("updatedAt", startInstant.plusMillis(2000).toEpochMilli())
                         .build()))
                 .add(new ProducerRecord<>(HYBRID_TABLE_NAME, "key3", new GenericRecordBuilder(hybridAvroSchema)
                         .set("stringCol", "string_3")
                         .set("longCol", 3L)
-                        .set("updatedAt", initialUpdatedAt.plusMillis(3000).toEpochMilli())
-                        .build()))
-                .add(new ProducerRecord<>(HYBRID_TABLE_NAME, "key4", new GenericRecordBuilder(hybridAvroSchema)
-                        .set("stringCol", "string_4")
-                        .set("longCol", 0L)
-                        .set("updatedAt", initialUpdatedAt.truncatedTo(DAYS).minusSeconds(3600).toEpochMilli())
-                        .build()))
-                .add(new ProducerRecord<>(HYBRID_TABLE_NAME, "key5", new GenericRecordBuilder(hybridAvroSchema)
-                        .set("stringCol", "string_5")
-                        .set("longCol", 1L)
-                        .set("updatedAt", initialUpdatedAt.truncatedTo(DAYS).minusSeconds(3600 * 2).toEpochMilli())
-                        .build()))
-                .add(new ProducerRecord<>(HYBRID_TABLE_NAME, "key6", new GenericRecordBuilder(hybridAvroSchema)
-                        .set("stringCol", "string_6")
-                        .set("longCol", 2L)
-                        .set("updatedAt", initialUpdatedAt.truncatedTo(DAYS).minusSeconds(86400 * 2 - 3600).toEpochMilli())
-                        .build()))
-                .add(new ProducerRecord<>(HYBRID_TABLE_NAME, "key7", new GenericRecordBuilder(hybridAvroSchema)
-                        .set("stringCol", "string_7")
-                        .set("longCol", 3L)
-                        .set("updatedAt", initialUpdatedAt.truncatedTo(DAYS).minusSeconds(86400 * 2 - 7200).toEpochMilli())
+                        .set("updatedAt", startInstant.plusMillis(3000).toEpochMilli())
                         .build()))
                 .build();
 
         Path temporaryDirectory = Paths.get("/tmp/segments-" + randomUUID());
         try {
             Files.createDirectory(temporaryDirectory);
             ImmutableList.Builder<GenericRow> offlineRowsBuilder = ImmutableList.builder();
-            for (int i = 8; i < 12; i++) {
+            for (int i = 4; i < 8; i++) {
                 GenericRow row = new GenericRow();
                 row.putValue("stringCol", "string_" + i);
-                row.putValue("longCol", (long) i - 8);
-                row.putValue("updatedAt", initialUpdatedAt.truncatedTo(DAYS).minusSeconds(3600 * 12 * (i - 8)).toEpochMilli());
+                row.putValue("longCol", (long) i);
+                row.putValue("updatedAt", startInstant.plusMillis(86_400_000 + 1000 * (i - 4)).toEpochMilli());
```

Comment:
How about `startInstant.plus(1, DAYS).plusMillis(1000 * (i - 4)).toEpochMilli()`?  Same for other places.